Dockerfile

```diff
@@ -1,17 +1,16 @@
 # Container image that runs your code
 FROM docker.io/snyk/snyk:linux@sha256:91a21f9dfd09a93126d6b066f86f995013148253d98284f300a7b69c12e1829c as snyk
+FROM docker.io/openpolicyagent/conftest:latest@sha256:93c0fccb97538b24ecb8492c5c988f39d27ec7108d4ce99f217677ad50f4271e as conftest
```

Comment:
I see we specified arch in previous approach, how are we specifying arch in this case?

Reply:
arch specific container will be used by default, if we build on x86_64, then x86_64 container will be used

Reply:
Why we don't use specific tag for specific release instead of latest? Using latest is usually antipattern.

https://hub.docker.com/layers/openpolicyagent/conftest/v0.44.1/images/sha256-93c0fccb97538b24ecb8492c5c988f39d27ec7108d4ce99f217677ad50f4271e?context=explore